src/VirtualClient/VirtualClient.Actions/HPLinpack/HPLinpackExecutor.cs

```diff
@@ -209,15 +209,24 @@ protected override async Task ExecuteAsync(EventContext telemetryContext, Cancel
         {
             using (BackgroundOperations profiling = BackgroundOperations.BeginProfiling(this, cancellationToken))
             {
+                IProcessProxy process;
+
                 DateTime startTime = DateTime.UtcNow;
-                await this.ExecuteCommandAsync("make", $"arch=Linux_GCC", this.HPLDirectory, telemetryContext, cancellationToken)
-                    .ConfigureAwait(false);
+                using (process = await this.ExecuteCommandAsync("make", $"arch=Linux_GCC", this.HPLDirectory, telemetryContext, cancellationToken).ConfigureAwait(false))
+                {
+                    if (!cancellationToken.IsCancellationRequested)
+                    {
+                        if (process.IsErrored())
+                        {
+                            await this.LogProcessDetailsAsync(process, telemetryContext, "HPLinpack", logToFile: true);
```

Comment:
the logProcessDetails should be outside of the process.IsErrored, also someworkloads should already be logging process details, see if there are duplications